app/views/content/subjects/engineers-teach-physics/_article.html.erb

```diff
@@ -0,0 +1,120 @@
+
+<article>
+  <section class="statement">
+    <div>
+      <span class="pink xlarge bold">Physics initial teacher training courses for</span>
+    </div>
+
+    <div>
+      <span class="pink medium bold">
+        <ul class="slashed-list">
+          <li>engineers</li>
```

Comment:
"engineers / material scientists" is possibly a little lost due to the formatting of this section (I know that this is the template component but it is a bit more important on this page I think?)

@gazcarr do you have a view on this?